docs/sections/workflows.html

```diff
@@ -0,0 +1,34 @@
+
+    <section>
+      <h2>Workflows</h2>
+      
+      <h3>Business Card Workflow</h3>
+      <image style='border: 1px solid #9a9a9a; border-radius: 3px; background-color: #2a303c;' src='resources/wf_businesscard.svg'/>
+      <p>This is a simple as possible workflow that describes an interaction between  two individuals. Alice and Bob are two individuals who support sending  presentations using the Traceability API. Alice issues a credential and  sends it to Bob in a Traceable Presentation.</p>
+      <b>Credentials Used:</b>
+      <ol>
+        <li><a href="undefined">Verifiable Business Card</a></li>
```

Comment:
why undefined?

Reply:
i changed from `url` to `id`, I can update this.